src/data.js

```diff
@@ -979,18 +979,19 @@ Worker.expire = async function(now) {
 };
 
 Worker.prototype.json = function() {
-  return {
+  let w = {
```

Comment:
I would rename to `const worker = { ... }` to make the variable name descriptive.